attest/index.js

```diff
@@ -107,6 +113,22 @@ async function run(config) {
     if (config.cimon.logLevel !== '')
         args.push('--log-level', config.cimon.logLevel);
     if (config.report.reportJobSummary) args.push('--report-job-summary');
+    if (config.attest.allowKeyless) {
+        args.push('--keyless');
+        
+        args.push(`--allow-tlog=${config.attest.allowTLogUpload}`);
+        args.push(`--allow-timestamp=${config.attest.allowTimestampUpload}`);
+        args.push(`--fulcio-server-url=${config.attest.fulcioServerUrl}`);
+
+        if (config.attest.allowTLogUpload) {
+            args.push(`--rekor-server-url=${config.attest.rekorServerUrl}`);
+        }
+        if (config.attest.allowTimestampUpload && config.attest.timestampServerUrl !== '') {
+            args.push(`--timestamp-server-url=${config.attest.timestampServerUrl}`);
+        } else if (config.attest.allowTimestampUpload) {
+            throw new Error('Timestamp server URL is required when allow-timestamp-upload is true');
+        }
+    }
```

Comment:
Let's keep these actions free of such logics. It is unbearable to maintain such logic in actions for every CI.
Let's just pass the params, and let the CLI to take care of it